Azure/sonic-utilities
# [show]: split show interfaces commands into separate file

Signed-off-by: Guohan Lu <lguohan@gmail.com>



**- What I did**
[show]: split show interfaces commands into separate file

add extensive test coverage for show interfaces commands.

**- How I did it**

**- How to verify it**
unit test and on box test

unit test coverage

```
show/interfaces.py                           223     73    67%
```

a few things not covered.

show breakout (need to check with @zxu why it is not covered before)
show interfaces lpmode (i think this should be add by part of platform efforts)
show interfaces counters ... (this has never been added, need new efforts to add this)

all others commands are almost 100% covered by the unit tests.

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**
